web/src/components/suggestion/SuggestionEvent.vue

```diff
@@ -8,30 +8,73 @@
       <div class="event-info">
         <p class="event-user">
           <span class="user-name">{{ user.name }} </span>
-          <span v-if="type == 'ACTION'">
+          <span v-if="type === eventTypes.ACTION">
             {{ event.text }}
             <span class="tag">{{ event.value }}</span>
           </span>
         </p>
         <p class="date-sent">{{ dateTimeFormatLabel(this.event.created) }}</p>
       </div>
+      <div
+        v-if="isAuthenticated && (role === userRoles.ADMIN || authedUserId === event.user_id)"
+        class="menu-wrapper">
+        <menu-button
+          v-if="type === eventTypes.COMMENT"
+          :options="commentOptions"
+          ref="menu"
+          class="menu" />
+        <menu-button
+          v-if="type === eventTypes.ACTION"
+          :options="actionOptions"
+          ref="menu"
+          class="menu" />
+      </div>
     </div>
-    <div v-if="type == 'COMMENT'" class="event-comment">
-      <p>{{ event.text }}</p>
+    <div v-if="type === eventTypes.COMMENT">
+        <div v-if="!isEditable" class="event-comment">
+          <p v-html="$sanitize(content)"></p>
+        </div>
+        <div v-show="isEditable" class="edit-comment">
+          <markdown-editor
+            v-model="event.text"
+            ref="eventMarkdownEditor"
+            :configs="mdeConfigs">
+          </markdown-editor>
+          <div class="comment-submit">
+            <span @click="saveComment" class="submit-button">
+              Tallenna
+            </span>
+          </div>
+        </div>
     </div>
   </div>
 </div>
 </template>
 
 <script>
+import MenuButton from '../common/MenuButton';
+import markdownEditor from 'vue-simplemde/src/markdown-editor';
 import { dateTimeFormatLabel } from '../../utils/dateHelper';
+import { userRoles } from '../../utils/userHelpers';
+
+// eslint-disable-next-line
+import { mapAuthenticatedUserGetters } from '../../store/modules/authenticatedUser/authenticatedUserModule.js';
+// eslint-disable-next-line
+import { authenticatedUserGetters } from '../../store/modules/authenticatedUser/authenticatedUserConsts.js';
 
+import { eventTypes } from '../../utils/eventHelper.js';
```

Comment:
kantsii yhdistää näitä importteja tossa alempana toi eventhelper importti kans

Reply:
jesh